chromium_src/components/search_engines/template_url_prepopulate_data.cc

```diff
@@ -186,6 +368,23 @@ GetBravePrepopulatedEnginesForCountryID(
 
 }  // namespace
 
+// Some engines (like Yahoo) have different URLs per country
+// The intention of this function is to find the generic one
+// (ex: PREPOPULATED_ENGINE_ID_YAHOO) and then substitute the
+// country specific version.
+void LocalizeEngineList(
+    std::vector<BravePrepopulatedEngineID>* engines,
```

Comment:
just pass this by const reference

Reply:
sorry, you apparently modify this vector. In this case it probably should be the last argument, as CC prefers putting "input" parameters before "output" ones.

Reply:
OK will update to last param 😄 good catch